graphql/schema.graphql

```diff
@@ -155,5 +156,9 @@ type File {
   visibility: String!
 }
 
+type User {
```

Comment:
Instead of defining a separate type, can you create an APIUser struct in the rest/model package, and map it to this type in the gqlgen.yml file? The reasoning for that is that we'll almost certainly need to add fields from the user rest model to this in the future, and those already have converters for the db models

Reply:
APIAuthUser is actually the wrong one to use - that's a data structure used to define static users for the naive auth system. I meant to create a new struct that can just have the display name field for now